Dockerfile

```diff
@@ -4,14 +4,23 @@ FROM docker.io/usercont/base
 
 ENV LC_ALL=C \
     PYTHONDONTWRITEBYTECODE=yes \
-    WORKDIR=/src
+    WORKDIR=/src \
+    USER=sandcastle \
+    HOME=/home/sandcastle
+
+# npm needs to access ~/.npm and ~/.config
+RUN mkdir --mode 0776 ${HOME}
```

Comment:
this should ideally be owned by sandcastle user

Reply:
there's no such user at the time of building the image